base/loading.jl

```diff
@@ -341,7 +337,7 @@ during package development as code is modified.
 function reload(name::AbstractString)
     if isfile(name) || contains(name,Filesystem.path_separator)
         # for reload("path/file.jl") just ask for include instead
-        error("use `include` instead of `reload` to load source files")
+        error("Use `include` instead of `reload` to load source files.")
```

Comment:
I think that we should use standard English capitalization for any error message that is written out as a complete sentence, but not for error messages that are sentence fragments.

e.g. `error("The foo needs to be frobnicated.")` vs. `error("foo not frobnicated")`.  Of course, we could discourage sentence fragments in error messages, but I'm inclined to think that there is a place for abbreviated messages.